src/modules/core/components/AvatarUploader/AvatarUploader.css

```diff
@@ -6,6 +6,43 @@
   border-radius: 50%;
 }
 
+.dropzoneNoButtonsVariant {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 77px;
+  position: relative;
+  overflow: hidden;
+  background-color: rgba(200, 214, 245, 0.33);
```

Comment:
You could use `background-color: color-mod(var(----temp-grey-blue-4) alpha(33%));` here